src/Template/Bake/Plugin/composer.json.ctp

```diff
@@ -20,7 +20,7 @@ $namespace = str_replace('\\', '\\\\', $namespace);
     "type": "cakephp-plugin",
     "require": {
         "php": ">=5.5.9",
-        "cakephp/cakephp": "3.3.*"
+        "cakephp/cakephp": ">=3.3.0 <4.0.0"
```

Comment:
Also given how was the 3.3.0 release we might better recommend 3.3.2 what do you think